README.md

```diff
@@ -106,6 +106,25 @@ jobs:
       uses: actions/checkout@v2
 ```
 
+#### `total length of command line and environment variables exceeds limit`
+
+If the error `total length of command line and environment variables exceeds limit` appears, then
+the current environment variables' total size has exceeded the maximum when executing Go Wasm binaries.
+
+To resolve this issue, install `cleanenv` and use it to prefix your command.
+
+For example, if this is the failing line in GitHub Actions:
```

Comment:
For example, if this is the command used:

Reply:
Fixed in 6a93001